include/vcpkg/base/messages.h

```diff
@@ -1580,6 +1580,7 @@ namespace vcpkg
                     "'--' at the beginning must be preserved",
                     "-- Using community triplet {triplet}. This triplet configuration is not guaranteed to succeed.");
     DECLARE_MESSAGE(UsingManifestAt, (msg::path), "", "Using manifest file at {path}.");
+    DECLARE_MESSAGE(Utf8DecoderDereferencedAtEof, (), "", "dereferenced Utf8Decoder at the end of a string.");
```

Comment:
This is a program bug rather than a user message so I don't think it should be localized. (Maybe in all such cases we should be outright deleting the message and adding a comment to the line instead? /cc @JavierMatosD  )